app/controllers/admin/partners_controller.rb

```diff
@@ -144,6 +144,12 @@ def clear_address
       end
     end
 
+    def lookup_name
+      found = params[:name].present? && Partner.where(name: params[:name]).first
```

Comment:
I'm able to type `caracol` without a warning but I get a rejection because `Caracol` already exists, is there a way to check for partners that isn't case sensitive.

Reply:
fixed